src/stripe.common.ts

```diff
@@ -0,0 +1,481 @@
+import _Error = require('./Error');
+import http = require('http');
+
+import resources = require('./resources');
+
+const DEFAULT_HOST = 'api.stripe.com';
+const DEFAULT_PORT = '443';
+const DEFAULT_BASE_PATH = '/v1/';
+const DEFAULT_API_VERSION = (null as unknown) as string;
+
+const DEFAULT_TIMEOUT = 80000;
+
+Stripe.PACKAGE_VERSION = require('../package.json').version;
+
+const determineProcessUserAgentProperties = (): Record<string, string> => {
+  return typeof process === 'undefined'
```

Comment:
Looks shady. Worth extracting?

Reply:
Agreed, added to list of things to investigate after this PR